lib/testing/test_runner.sh

```diff
@@ -102,6 +121,13 @@ skip_if_legacy_fallback()
     fi
 }
 
+skip_if_missing_trace_attach()
+{
+    if ! is_trace_attach_supported; then
```

Comment:
This is called multiple times; should we cache the result?

Reply:
Yes, not sure why I removed that part, it was originally there as IO coped the below.